browser/pages/account/controller.js

```diff
@@ -14,12 +19,17 @@ class AccountController {
     this.installerDataSvc = installerDataSvc;
     this.electron = electron;
     $scope.version = electron.remote.app.getVersion();
-    this.username = '';
-    this.password = '';
     this.authFailed = false;
     this.tandcNotSigned = false;
     this.isLoginBtnClicked = false;
     this.httpError = undefined;
+    $scope.credential = TokenStore;
+    this.username = $scope.credential.TokenStore.getUserName();
+    let password = $scope.credential.TokenStore.getItem('login', this.username);
+    password.then((pass) => {
```

Comment:
getItem returns Promise instance which can be fulfilled when user entering username and that would be confusing. I suggest you to do this kind of initialization in InstallerDataService constructor for now.
Then just:
``` JavaScript
this.username = this.installerDataSvc.username;
this.password= this.installerDataSvc.password;
```